### DISCUSSION TITLE
Introduction - Shubham

### DISCUSSION THREAD

**Discussion by shubhamdo** [aiqc/AIQC#85] [Introduction]
Hey guys, my name is **Shubham** and I'm from **Germany**. 

I'm a _Data Engineer/Analyst_ at the moment, and work mostly with ETL Tools,
Visualization tools like Tableau, SSIS etc. Pursuing Master in Data Science parallelly, I try to actively contribute to opensource, have contributed to some data science python packages like scikit.

_Tech interests:_ NLP, Deep Learning, NoSQL Database, APIs, etc.
_Life interests:_ video games, photography, motor sport, movies

**Comment by aiqc** [comment]
RE: motor sport, the Formula1 Netflix documentary is getting a lot of Americans interested in racing! Do you follow the Bundesliga at all?

**Comment by Bennylave** [comment]
Welcome to the team Shubham! I'm sure you'll be a great fit
